### ISSUE TITLE
Toggle LED On when WAV is played

### ISSUE THREAD

**Issue by sadmin91** [project-alice-assistant/HermesLedControl#103]
**Is your feature request related to a problem? Please describe.**
No LED are turn on when I play a WAV file on my Rhasspy instance by calling play-wav endpoint API

**Describe the solution you'd like**
Activate LEDs when a WAV file is played on my Rhasspy instance

**Comment by Psychokiller1888** [comment]
Hi

What's the mqtt topic sent when playing a wav file?

**Comment by sadmin91** [comment]
Hello,
By reading the Rhasspy documentation (https://rhasspy.readthedocs.io/en/latest/reference/#audio-server), I think that the mqtt topic is `hermes/audioServer/<siteId>/playBytes/<requestId>`

**Comment by Psychokiller1888** [comment]
No, playBytes is not connected to HLC, you should use the higher API of Hermes hermes/tts/say
